backend/de.metas.payment.esr/src/main/java/de/metas/payment/esr/api/impl/ESRImportBL.java

```diff
@@ -147,6 +147,7 @@ private final void lockAndProcess(
 			@NonNull final I_ESR_Import esrImport,
 			@NonNull final Runnable processor)
 	{
+		trxManager.assertThreadInheritedTrxNotExists();
```

Comment:
there is also a process which calls this.
make sure that process runs out of trx too.